mm2src/coins/eth/eip1559_gas_fee.rs

```diff
@@ -0,0 +1,469 @@
+use super::web3_transport::{EthFeeHistoryNamespace, FeeHistoryResult};
+use super::{u256_to_big_decimal, Web3RpcError, Web3RpcResult, ETH_GWEI_DECIMALS};
+use common::log::info;
+use ethereum_types::U256;
+use http::StatusCode;
+use mm2_err_handle::mm_error::MmError;
+use mm2_err_handle::prelude::*;
+use mm2_net::transport::slurp_url_with_headers;
+use mm2_number::BigDecimal;
+use num_traits::FromPrimitive;
+use serde::de::{Deserializer, SeqAccess, Visitor};
+use serde_json::{self as json};
+use std::collections::HashMap;
+use std::fmt;
+use web3::{types::BlockNumber, Transport};
+
+// Estimate base and priority fee per gas
+// or fetch estimations from a gas api provider
```

Comment:
What is the purpose of this orphan comment?

Reply:
made a source file doc comment instead